crates/astria-account-monitor/src/config.rs

```diff
@@ -0,0 +1,103 @@
+use astria_eyre::eyre;
+use sequencer_client::Address;
+use serde::{
+    Deserialize,
+    Serialize,
+};
+
+#[derive(Debug, Deserialize, Serialize)]
+/// The high-level config for creating an astria-account-monitor service.
+pub struct Config {
+    /// Log level. One of debug, info, warn, or error
+    pub log: String,
+
+    /// Address of the ABCI server for the sequencer chain
+    pub sequencer_abci_endpoint: String,
+
+    /// The chain ID of the sequencer chain
+    pub sequencer_chain_id: String,
+
+    /// The address prefix to use when constructing sequencer addresses using the signing key.
+    pub sequencer_address_prefix: String,
+
+    /// The addresses of the sequencer chain to monitor.
+    pub sequencer_accounts: String,
+
+    pub sequencer_bridge_accounts: String,
+    /// The asset ID of the sequencer chain to monitor.
+    pub sequencer_asset: String,
+
+    /// Sequencer block time in milliseconds
+    #[serde(alias = "query_interval_ms")]
+    pub block_time_ms: u64,
+
+    /// Forces writing trace data to stdout no matter if connected to a tty or not.
+    pub force_stdout: bool,
+
+    /// Disables writing trace data to an opentelemetry endpoint.
+    pub no_otel: bool,
+
+    /// Set to true to disable the metrics server
+    pub no_metrics: bool,
```

Comment:
note: I know we do this everywhere else, but I think it might not make sense to run this binary at all with the metrics server disabled?